self-healing/transform-ldes-data-to-entities.ts

```diff
@@ -101,15 +110,66 @@ export async function transformRemainingEntities() {
       }
     }
     WHERE {
+      { SELECT ?ldesEntity WHERE {
+          GRAPH ${sparqlEscapeUri(HEALING_DUMP_GRAPH)} {
+            ?ldesStream tree:member ?ldesEntity.
+            ?ldesEntity prov:generatedAtTime ?time.
+          }
+      } LIMIT 10000 }
       GRAPH ${sparqlEscapeUri(HEALING_DUMP_GRAPH)} {
-        ?ldesStream tree:member ?ldesEntity.
-        ?ldesEntity ?p ?o.
         ?ldesEntity dct:isVersionOf ?entity.
-        ?ldesEntity prov:generatedAtTime ?time.
+        ?ldesEntity ?p ?o.
```

Comment:
Just copying over everything does not make it more reliable, why is this?

Reply:
the git diff is confusing here. The ?p ?o statement was there before. BUT the limit was added AND it's inside a subquery so the predicates aren't limited, only the instances (else you risk breaking predicates of an instance because of a cutoff by the limit)